ant-networking/src/lib.rs

```diff
@@ -1075,6 +1075,7 @@ impl Network {
         &self,
         key: &NetworkAddress,
         client: bool,
+        expanded_close_group_size: Option<usize>,
```

Comment:
I did not want unintentional side effects on the node side for increasing the close group size here, thus an option.